src/Adapter/Product/QueryHandler/SearchProductsHandler.php

```diff
@@ -197,7 +197,7 @@ private function createFoundProductFromLegacy(
             $this->tools->round($priceTaxExcluded, $computingPrecision),
             $product->getTaxesRate($address),
             Product::getQuantity($product->id),
-            $product->location,
+            $product->location ?: '',
```

Comment:
I think this should be fixed in `Product::loadStockData` instead It will fix more use cases